llvm/lib/Target/SPIRV/SPIRVSubtarget.cpp

```diff
@@ -11,6 +11,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "SPIRVSubtarget.h"
+#include "MCTargetDesc/SPIRVBaseInfo.h"
```

Comment:
Do we really need this?

Reply:
Added this initially for another reason and it's not needed any longer. Thanks.